assignments/assignment01.rb

```diff
@@ -38,7 +40,25 @@ def number_to_string(n, lang)
 # it accepts a string
 # and returns the same string with each word capitalized.
 def titleize(s)
-  # your implementation here
+
+  # => PSEUDO CODE
+  # => -----------------------------------------------------------------
+  # => downcase all characters, then capitalize first character
+  # => iterate the string, capitalize each first letter every after space
+  # => save each change to new string
+  # => return the string
+  # => ------------------------------------------------------------------
+
+  s = s.downcase.capitalize
+  count = 0
+  while count < s.length
+    if s[count] == " "
+      s[count + 1] = s[count + 1].capitalize
+    end
+    count += 1
+  end
+  print s
```

Comment:
you don't want to print this out. this might be useful during debugging, but you don't want your final solution to have the side-effect of print something out.